### DISCUSSION TITLE
[教程]多设备同步设置方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2991] [新手建议]
# 重要提示

> ### 嗨，简悦全新的 [教程中心](https://www.yuque.com/kenshin/simpread) 已上线，**下面的内容并不一定是最新的**，建议 [访问这里](https://www.yuque.com/kenshin/simpread/dgmm5q) 查看最新版教程。

***

> 以下内容来自 简悦 **[官方 Telegram 群](https://t.me/simpread)** 用户 [DavidJoy](https://github.com/DavidJoyDong) 的方案（从用户视角的配置方案）

# 主电脑
简悦设置好后，使用 **同步助手把配置文件保存在坚果云的同步文件夹**中。

# 第二台电脑

> 安装坚果云、简悦、同步助手。
  
1. 坚果云设置好同步文件夹  
2. 同步助手设置到同步文件夹  
  
进入简悦设置，手动导入一次配置文件（导入了高级账户，同步助手开始运转），再在 **插件管理** 手动从配置文件中导入简悦的插件，基本上和主电脑一致了。

# 安卓手机

> 安装Kiwi、简悦

不用管简悦账户的事，阅读模式直接用就行了，想导出就去设置里单独授权

## 补充

> 除了上述方案外，也可以使用 https://github.com/Kenshin/simpread/discussions/1574 提供的登录方式。

## 关联

- [高级账户相关说明（包括：绑定 · 授权 · 找回 UID · 被锁定的处理等） #1577](https://github.com/Kenshin/simpread/discussions/1577)
- [[新手教程\]如何安装和配置同步助手 #2754](https://github.com/Kenshin/simpread/discussions/2754)
- [登录（在新设备如何使用简悦的高级账户） #1574](https://github.com/Kenshin/simpread/discussions/1574)

## 引申

- [简悦多种同步方案对比（建议有同步需求的用户都需要看看） #3138](https://github.com/Kenshin/simpread/discussions/3138)

**Comment by DavidJoyDong** [comment]
这个是只使用同步助手而没有授权坚果云的同步配置方法，授权坚果云后就更简单了

**Comment by DavidJoyDong** [comment]
补充一下，这个功能需要开
![image](https://user-images.githubusercontent.com/47232992/144461102-a37d05a6-ba13-4026-9445-218897b7be0a.png)
